x/gamm/pool-models/stableswap/pool_test.go

```diff
@@ -0,0 +1,118 @@
+package stableswap
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/suite"
+
+	"github.com/osmosis-labs/osmosis/v11/x/gamm/pool-models/internal/test_helpers"
+)
+
+type StableSwapTestSuite struct {
+	test_helpers.CfmmCommonTestSuite
+}
+
+func TestStableSwapTestSuite(t *testing.T) {
+	suite.Run(t, new(StableSwapTestSuite))
+}
+
+// func (suite *StableSwapTestSuite) Test_StableSwap_CalculateAmountOutAndIn_InverseRelationship() {
```

Comment:
Why was this commented out?

Reply:
because, stableswap math is not ready for testing, I think

Reply:
sorry for the delay in review here, does this test not pass if we comment this out? (Asking this seeing that it wasn't commented out originally)

Reply:
This test wasn't commented out before but it still did not run.

Reply:
Hrrm what's the reason it didn't run? Sorry if Im missing something here